src/ws-tiles-chart/tile.js

```diff
@@ -0,0 +1,21 @@
+import {Component, PropTypes} from "../imports";
```

Comment:
Could you please also import React here? The jsx transpiles to React.createElement and if you don't add React to the window scope this won't work.